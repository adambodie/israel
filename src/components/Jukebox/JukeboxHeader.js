import React from 'react'

const JukeboxHeader = () => (
	<div className='jukebox-title'>
		<h1 data-testid='title-display'>Sababa!</h1>
	</div>
)

export default JukeboxHeader
