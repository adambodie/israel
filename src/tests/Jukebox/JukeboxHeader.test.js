import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import JukeboxHeader from '../../components/Jukebox/JukeboxHeader'
import { cleanup } from 'react-testing-library'
  
afterEach(cleanup)

it('renders without crashing', () => {
	const div = document.createElement('div')
	ReactDOM.render(<JukeboxHeader />, div)
	ReactDOM.unmountComponentAtNode(div)
})

it('renders correctly', () => {
	const tree = renderer.create(<JukeboxHeader />).toJSON()
	expect(tree).toMatchSnapshot()
})