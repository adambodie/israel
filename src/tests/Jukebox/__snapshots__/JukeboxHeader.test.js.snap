// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="jukebox-title"
>
  <h1
    data-testid="title-display"
  >
    Sababa!
  </h1>
</div>
`;
